chronopost.php

```diff
@@ -875,9 +897,29 @@ public function getOrderShippingCost($cart, $shipping_cost)
 			return $shipping_cost;
 		}
 
+		if ($res->return->amountTTC != 0)
+		{
+			if(empty($cache))
+			{
+				DB::getInstance()->query('INSERT INTO '._DB_PREFIX_.'chrono_quickcost_cache (product_code, arrcode, weight, price, last_updated) VALUES (
+						"'.$productCode.'",
+						"'.$arrcode.'",
+						"'.$cart->getTotalWeight().'",
+						"'.$res->return->amountTTC.'",
```

Comment:
Can you protect a little bit more the data sent to your database ? with an `(int)` cast for numbers and `pSql(...)` for strings